### PR TITLE
Autogen bug fixes

### PR DESCRIPTION
**Pull request by pvienhage**
- [ ] Tag the PR with `wip` while in development.
- [ ] Assign yourself as to the PR
- [ ] Assign relevant labels such as `bug`, `enhancement`.
- [ ] Request reviews if the PR is large, complex or you would like an extra pair of eyes to go over it.
- [ ] Add tests to cover changes as needed.
- [ ] Update documentation as needed.
- [ ] Add new entries to the `Changelog.md`.
- [ ] Update version numbers as needed.

Semver: https://github.com/rust-lang/rfcs/blob/master/text/1105-api-evolution.md

https://semver.org/

### CODE REVIEW

File: crypto/stark/assets/OodsPoly.sol [hunk 1]
```diff
@@ -65,13 +65,15 @@ contract OodsPoly \{
...
 
             function mid_expmod(base, exponent) -> result \{
-                result := 1
-                for \{  } exponent \{ exponent := shr(exponent, 1) } \{
-                    if and(exponent, 1) \{
-                        result := mulmod(result, base, {modulus})
-                    }
-                    base := mulmod(base, base, {modulus})
-                }
+                // TOOD - Find and fix the trivial bug
```
Comment (z2trillion) [inline crypto/stark/assets/OodsPoly.sol:68]: TOOD -> TODO

File: crypto/stark/assets/OodsPoly.sol [hunk 2]
```diff
@@ -65,13 +65,15 @@ contract OodsPoly \{
...
+                // result := 1
+                // for \{  } exponent \{ exponent := shr(exponent, 1) } \{
+                //     if and(exponent, 1) \{
+                //         result := mulmod(result, base, {modulus})
+                //     }
+                //     base := mulmod(base, base, {modulus})
+                // }
+                result := expmod(base, exponent)
             }
```
Comment (Recmo) [inline crypto/stark/assets/OodsPoly.sol:77]: Let's just remove the code. Also, there should be a corresponding change in the generator to stop using it.

**Review by Recmo — CHANGES_REQUESTED** [review]
Missing code changes in the invocation of the template?

File: crypto/stark/assets/OodsPoly.sol (hunk 2 above)
Comment (Recmo) [reply crypto/stark/assets/OodsPoly.sol:77]: i.e. here: https://github.com/0xProject/OpenZKP/blob/master/crypto/stark/src/solidity_verifier.rs#L179

**Review by Recmo — APPROVED** [review]

**Pull request merged**